index.js

```diff
@@ -46,5 +46,31 @@ server.get('/api/posts/:id', (req, res) => {
         });
 });
 
+// handles post requests
+server.post('/api/posts', (req, res) => {
+    const post = req.body;
+    db.insert(post)
+        .then(post => {
+            if (post.title && post.contents) {
+                res
+                    .status(201)
+                    .json({ post })
+            } else {
+                res
+                    .status(400)
+                    .json({ 
+                        errorMessage: "Please provide title and contents for the post." 
+                    })
+            }
+        })
+        .catch(error => {
```

Comment:
Why pass a parameter here when you don't use it anywhere? There could be an argument for this just making it easier later if you need this, but it is a bit of a hamper on readability if you don't have syntax highlighting for this sort of thing.